provider.go

```diff
@@ -185,3 +185,31 @@ func (m ProviderModule) postProviderHandler(c *gin.Context) {
 
 	c.JSON(http.StatusCreated, provider)
 }
+
+// putProviderHandler godoc
+// @summary Put provider in database.
+// @description Creates a new provider if a match is not found.
+// @tags provider
+// @accept json
+// @produce json
+// @param provider body Provider _ "provider object"
+// @success 200 {object} Provider
+// @failure 400 {object} problem.Response "Bad request"
+// @failure 401 {object} problem.Response "Unauthorized or missing jwt token"
+// @failure 502 {object} problem.Response "Database is unreachable"
+// @router /provider [put]
+func (m ProviderModule) putProviderHandler(c *gin.Context) {
```

Comment:
Forgotten to add this method to the `ProviderModule.Register` method

Reply:
Its in!
https://github.com/iver-wharf/wharf-api/blame/63c2c4986a42f58394f8db8729265fe5ba154a6d/provider.go#L42